material-ui/material-ui.scss

```diff
@@ -120,3 +121,120 @@ $swal2-close-button-hover-color: $swal2-close-button-color;
     }
   }
 }
+
+// Checkbox
+.swal2-checkbox {
+  input {
+    position: relative;
+    left: 30px;
+    width: 40px;
+    height: 40px;
+    margin: 0;
+    padding: 20px;
+    transform: scale(1);
+    transition: opacity .3s, transform .2s;
+    border-radius: 50%;
+    outline: none;
+    background-color: $material-ui-blue;
+    box-shadow: none;
+    opacity: 0;
+    z-index: -1;
+    appearance: none;
+    pointer-events: none;
+
+    &:checked,
+    &:indeterminate {
+      background-color: $material-ui-blue;
+    }
+
+    &:checked + span {
+      &::before {
+        border-color: $material-ui-blue;
+        background-color: $material-ui-blue;
+      }
+
+      &::after {
+        border-color: $white;
+      }
+    }
+
+    &:indeterminate + span {
+      &::before {
+        border-color: $material-ui-blue;
+        background-color: $material-ui-blue;
+      }
+
+      &::after {
+        transform: translate(4px, 3px);
```

Comment:
Could these two values be calculated based on a variable? Otherwise they may be difficult to maintain moving on. For example, if they are related to the size of the checkbox square, we may want to define that as variable and derive these two dimensions from there.

Reply:
I was thinking to create two variables like this:

$ckbx-align: 4;
$ckbx-type: 'px';

then:

transform: translate($ckbx-align +$ckbx-type, $ckbx-align-1 + $ckbx-type);

Is this what you had in mind?

Thanks @gverni

Reply:
I've done a bit of testing with my idea @Yairsaenz7, but it actually becomes quite messy 😞 so I'm thinking to drop it completely (see more below)

Still I think something can be done to have more variables. For example, how about introducing the following variables: 

```
$checkbox-size-px: 18;
$checkbox-left-px: -2;
```

and then use them this way for the box: 

```
 span {
    &::before {
      position: relative;
      top: 1px;
      left: $checkbox-left-px * 1px;
      width: $checkbox-size-px * 1px;
      height: $checkbox-size-px * 1px;
```
and for the check:
```
    &::after {
      position: relative;
      left: (-$checkbox-size-px + $checkbox-left-px) * 1px;
```

What do you think? 

----

FYI the formula to compute the translation was (applying some notion of trigonometry) something like : 
```
translate(($checkbox-size - (10+5)* .7)/2, ($checkbox-size - (10+2)*.7)/2))
```
where `$checkbox-size` is `18`px. Unfortunately that didn't really aligned the check sign, so more changes were needed in other part of css.

Reply:
I think it is a good idea 👍. I will update the code accordingly.